tools/submission/submission-checker.py

```diff
@@ -650,6 +650,14 @@
     "disk_controllers"
 ]
 
+SYSTEM_DESC_IS_NETWORK_MODE = "is_network"
+SYSTEM_DESC_REQUIRED_FIELDS_NETWORK_MODE = [             # TODO define exact fields
+    SYSTEM_DESC_IS_NETWORK_MODE, "network_type", "network_rate", "nic_loadgen", "number_nic_loadgen",
+    "net_software_stack_loadgen", "network_protocol", "number_connections", "nic_sut", "number_nic_sut",
+    "net_software_stack_sut", "network_setup"
+]
+NETWORK_MODE_REQUIRED_SUBSTRING_IN_SUT_NAME = "Over the Network SUT"
+NETWORK_MODE_MIN_VALID_GET_SUT_NAME_LATENCY_MS = 1 # TODO define min valid latency
```

Comment:
Just logging. Remove check of the latency threshold

Reply:
done